beater/burrowbeat.go

```diff
@@ -57,7 +57,28 @@ func (bt *Burrowbeat) Run(b *beat.Beat) error {
 		case <-ticker.C:
 		}
 
-		for _, group := range bt.groups {
+		logp.Debug("main", "Running tick")
+		groups := bt.groups
+		if len(groups) == 0 {
+			endpoint := "http://" + bt.host + ":" + bt.port + "/v2/kafka/" + bt.cluster + "/consumer"
```

Comment:
Most of the endpoint is the same as below. Maybe create a variable like `endpoint_base` and then add the suffix as required when you use it.